packages/react-cosmos-voyager/src/match-fixture-path.js

```diff
@@ -2,7 +2,7 @@ import { FIXTURE_EXTENSIONS_REGEX } from './fixture-extensions';
 
 const buildPathMatchers = (componentName, fixturesDir) => [
   `/${fixturesDir}/${componentName}/([^/]+)\\.(${FIXTURE_EXTENSIONS_REGEX})$`,
-  `/${componentName}/(?:.+/)?${fixturesDir}/([^/]+)\\.(${FIXTURE_EXTENSIONS_REGEX})$`,
+  `/${componentName}/${fixturesDir}/([^/]+)\\.(${FIXTURE_EXTENSIONS_REGEX})$`,
```

Comment:
This wouldn't be very pretty, but what if we also added this rule?
```js
`/${componentName}/src/${fixturesDir}/([^/]+)\\.(${FIXTURE_EXTENSIONS_REGEX})$`,
```
This would keep the previous test passing (which is a real life scenario used by large code base using Cosmos), and still fix the problem. The only way this can backfire is if somebody names a component as lowercase "src", and nests it under another component. It's an edge case we can live with until we find a better solution to refactor this part.